app/controllers/concerns/phone_confirmation.rb

```diff
@@ -1,8 +1,10 @@
 module PhoneConfirmation
   def prompt_to_confirm_phone(id:, phone:, selected_delivery_method: nil,
-                              selected_default_number: nil)
+                              selected_default_number: nil, phone_type: nil)
+
     user_session[:unconfirmed_phone] = phone
     user_session[:context] = 'confirmation'
+    user_session[:phone_type] = phone_type.to_s
```

Comment:
Wondering about the `to_s`, I'm assuming it's because the incoming value would be a symbol? Sorta curious how this should be expected to work when it's `nil`, since `nil.to_s` would be an empty string, where a string would be set to the session, yet `nil` would effectively unset the value?

I wonder if it would work to just avoid the casting, since I expect it'd be serialized to a string anyways if it's a symbol. Maybe that's too opaque though.

```suggestion
    user_session[:phone_type] = phone_type
```